OpenRA.Mods.Common/Scripting/Global/TriggerGlobal.cs

```diff
@@ -51,6 +51,20 @@ public void AfterDelay(int delay, LuaFunction func)
 			Context.World.AddFrameEndTask(w => w.Add(new DelayedAction(delay, doCall)));
 		}
 
+		[Desc("Call a function for each passenger when it enters a transport. " +
+		      "The callback function will be called as func(Actor passenger).")]
```

Comment:
It might be useful to get the transport in the callback as well (e.g., make the transport move somewhere once an actor has entered). Same with `OnPassengerExited`.